src/fragment-components/a-list.tsx

```diff
@@ -0,0 +1,272 @@
+import React, { useEffect, useState } from 'react';
+import { HierarchyList } from 'carbon-addons-iot-react';
+import {
+	TextInput,
+	OrderedList,
+	Accordion,
+	AccordionItem,
+	Button
+} from 'carbon-components-react';
+import { AComponent, ComponentInfo } from './a-component';
+import { useFragment } from '../context';
+import { css, cx } from 'emotion';
+import {
+	Add16,
+	TrashCan16
+} from '@carbon/icons-react';
+import {
+	getParentComponent
+} from '../components';
+import image from './../assets/component-icons/list.svg';
+import {
+	angularClassNamesFromComponentObj,
+	reactClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+import { actionIconStyle } from '../routes';
+
+const LayoutStyle = css`
+	.iot--list--page {
+		display: none;
+	}
+`;
+const OrderedListStyle = css`
+	margin-left: 20px;
+	list-style: auto;
+`;
+
+const getComponentObjById = (id: string, componentObj: any) => {
+	if (componentObj.id === id) {
+		return componentObj;
+	}
+
+	if (!componentObj.items) {
+		return undefined;
+	}
+
+	for (const item of componentObj.items) {
+		const foundComponentObj: any = getComponentObjById(id, item);
+		if (foundComponentObj) {
+			return foundComponentObj;
+		}
+	}
+	return undefined;
+};
```

Comment:
this looks familiar... 🤔

Reply:
its the same code as the one we used for layout :)